### PR TITLE
Reduce bundle size by requiring lodash function explicitly

### PR DESCRIPTION
**Pull request by pyromaniac**

### CODE REVIEW

**Comment by turt2live** [comment]
Thanks!

**Comment by pyromaniac** [comment]
Wow, this was fast, thank you!

File: lib/Email.js [hunk 1]
```diff
@@ -3,7 +3,9 @@
  * For the full license information, please see the LICENSE file distributed with this package.
  */
 
-var _ = require('lodash');
+var cloneDeep = require('lodash/cloneDeep');
```
Comment (dempfi) [inline lib/Email.js:6]: Don't use `var` at all, use `let` and `const`. Prefer `const` over `let`.

Comment (turt2live) [reply lib/Email.js:6]: That's something on my end that should be addressed.

**Comment by pyromaniac** [comment]
@turt2live will you release a new version?

**Comment by turt2live** [comment]
@pyromaniac just did, sorry about that. Looks like my Jenkins server failed to do it for me :(

**Comment by pyromaniac** [comment]
Perfect, thank you!

**Pull request merged**